### PR TITLE
Add some NFT metrics docs

### PR DESCRIPTION
**Pull request by alresharin**
## Changes

Add description for the following metrics:
nft_trades_count
nft_whale_trades_count
nft_retail_trades_count

nft_trade_volume_usd
nft_retail_trade_volume_usd
nft_whale_trade_volume_usd

## Checklist

- [x] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

**Review by alexander-gr — APPROVED** [review]

File: src/docs/metrics/nft-trade-volume-usd/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+`weth`
+`echelon-prime`
+`multi-collateral-dai`
+`p-matic-network`
+`p-multi-collateral-dai`
+`p-tether`
+`p-usd-coin`
+`p-weth`
+`the-sandbox`
+`usd-coin`
```
[the remark this answers is not in the document]
Comment (alresharin) [reply src/docs/metrics/nft-trade-volume-usd/index.md]: Fixed

File: src/docs/metrics/nft-trade-volume-usd/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+[On-Chain Latency](/metrics/details/latency#on-chain-latency)
+
+---
+
+## Available Assets
+
+- For `nft_trade_volume_usd` : [available assets](https://api.santiment.net/graphiql?query=%7B%0A%20%20getMetric(metric%3A%20%22nft_trade_volume_usd%22)%20%7B%0A%20%20%20%20metadata%20%7B%0A%20%20%20%20%20%20availableSlugs%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)
+
+- For `nft_retail_trade_volume_usd` and `nft_whale_trade_volume_usd`:
+`ethereum`
```
Comment (IvanIvanoff) [inline src/docs/metrics/nft-trade-volume-usd/index.md:52]: Why don't you use the API example for all metrics?

Comment (alresharin) [reply src/docs/metrics/nft-trade-volume-usd/index.md:52]: Hm, Initially i used other NFT metrics articles as an example where the available assets (usually eth or eth/weth only) are hardcoded. Looks like query is more common way regarding a lot of other articles though. Let me fix the hardcoded lists then

Comment (alresharin) [reply src/docs/metrics/nft-trade-volume-usd/index.md]: Fixed

**Review by IvanIvanoff — APPROVED** [review]

**Pull request merged**